### PR TITLE
Build lessons learned & a return to GraalVersion 19.2.1

### PR DESCRIPTION
**Pull request by edwardcallahan**
GraalVM 19.3.x is not building. Return to 19.2.1.

Adds link in Readme to The Build. Additional requirements, ie clang, not gcc, to be noted there in doc, keeping DRY.

### CODE REVIEW

**Comment by edwardcallahan** [comment]
Adds `npm install` step for `paradox` target and ignore `protocols/*.zip`

File: README.md [hunk 1]
```diff
@@ -482,6 +482,8 @@ A custom Akka cluster sharding rebalancing strategy may be able to be used to tr
 
 By default, we build a GraalVM native image for Linux. This is done inside a Docker container and so can be done on any platform with Docker installed. We also generate Docker images containing the native image, so the native image can be run on any platform using Docker too.
 
+For further informatoin about setting up the build environment, see [The Cloudstate Build](https://cloudstate.io/docs/developer/thebuild.html#the-cloudstate-build) documentation.
```
Comment (ihostage) [inline README.md]:
```suggestion
For further information about setting up the build environment, see [The Cloudstate Build](https://cloudstate.io/docs/developer/thebuild.html#the-cloudstate-build) documentation.
```

Comment (viktorklang) [reply README.md]: Thanks @ihostage!

**Comment by viktorklang** [comment]
Thanks @edwardcallahan!

**Pull request merged**